src/dungeon_pokemon_attributes.c

```diff
@@ -218,15 +218,15 @@ bool8 CanSeeTeammate(Entity * pokemon)
 u8 GetMoveTypeForMonster(Entity *pokemon, Move *pokeMove)
 {
     if (pokeMove->id == MOVE_HIDDEN_POWER)
-        return pokemon->axObj.info->hiddenPower.hiddenPowerType;
-    else
-        return GetMoveType(pokeMove);
+        return GetEntInfo(pokemon)->hiddenPower.hiddenPowerType;
+        else
```

Comment:
The old indentation here was correct.

Reply:
Huh, that's a weird bug in coccinelle/spatch I think. Good catch.
(Pretty sure it shouldn't affect indentation with the options I used).